src/main/java/org/folio/marc4ld/service/marc2ld/mapper/mapper/OtherVersionMapper.java

```diff
@@ -29,7 +30,7 @@
 @RequiredArgsConstructor
 public class OtherVersionMapper implements Marc2ldMapper {
```

Comment:
MARC 760-787 are called Linking Entries (refer https://www.loc.gov/marc/bibliographic/bd760787.html)
So, let us rename this class to LinkingEntriesMapper. 
We will enhance this class with more marc fields 777, 780 etc later.

Reply:
Done